Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -903,6 +905,23 @@ Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::CSS::C
     return cssProperties;
 }
 
+Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<String>>> InspectorCSSAgent::getLoadedFonts()
+{
+    auto fontFamilyNames = JSON::ArrayOf<String>::create();
+    auto* domAgent = m_instrumentingAgents.persistentDOMAgent();
```

Comment:
there's no guarantee that this always exists, so we need to check it before using it
```cpp
if (auto* domAgent = m_instrumentingAgents.persistentDOMAgent()) {
    for (auto* document : domAgent->documents()) {
        auto it = document->fonts()->createIterator(document->scriptExecutionContext());
        while (auto font = it.next()) {
            if (font->status() == FontFace::LoadStatus::Loaded)
                fontFamilyNames->addItem(font->family());
        }
    }
}
```